eureka-cli/misc/docker-compose.yaml

```diff
@@ -175,10 +175,8 @@ services:
     deploy:
       resources:
         limits:
-          cpus: 1024m
           memory: 1200M
         reservations:
-          cpus: 1024m
```

Comment:
These values were errors on my environment: `error decoding 'services[keycloak-internal].deploy.resources.limits.cpus': strconv.ParseFloat: parsing "1024m": invalid syntax`

Reply:
Will test it

Reply:
Seems like these CPU properties are not supported by Docker engine, and maybe were accidently copy-pasted from some Helm chart